backend/src/main/java/gov/cdc/usds/simplereport/api/export/DataHubUploadController.java

```diff
@@ -48,7 +48,7 @@ public String uploadTestEventCSVToDataHub(@RequestParam String apikey,
         String headerKey = "Content-Disposition";
         String headerValue = "attachment; filename=testEvents_" + currentDateTime + ".csv";
         response.setHeader(headerKey, headerValue);
-        response.getWriter().print(_hubuploadservice.creatTestCVSForDataHub(startupdateby));
+        response.getWriter().print(_hubuploadservice.createTestCSVForDataHub(startupdateby));
```

Comment:
Note for #457 : this should return a `ResponseEntity<SomeObject>` where that object is what is returned by `createTestCSVForDataHub`, and it should use the builder to actually build that response entity instead of writing directly to the response object.